static/src/javascripts-legacy/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -44,8 +44,8 @@ define([
      * @type {{days: number, count: number}}
```

Comment:
As we're updating the default, could you correct its `@type` annotation, or remove it completely?